vendors/marvell/boards/mw300_rd/aws_demos/application_code/main.c

```diff
@@ -57,7 +57,7 @@ const AppVersion32_t xAppFirmwareVersion = {
 
 /* Logging Task Defines. */
 #define mainLOGGING_MESSAGE_QUEUE_LENGTH    ( 32 )
-#define mainLOGGING_TASK_STACK_SIZE         ( configMINIMAL_STACK_SIZE * 8 )
+#define mainLOGGING_TASK_STACK_SIZE         ( configMINIMAL_STACK_SIZE * 6 )
```

Comment:
Giving the logging task stack size in terms of multiples of minimal stack size is not helpful.  Just put in the number you want.  stack size is a set of educated guesses so lets not pretend this is the science to the method.

Reply:
Addressed it. Thanks.